tests/utils.py

```diff
@@ -1364,12 +1368,12 @@ def validate_atomic(opcode, recv_player, send_player, receiver_val,
                      swap.
     """
     send_expected = receiver_val
-    if opcode in [e.IBV_WR_ATOMIC_CMP_AND_SWP,
-                  e.IBV_QP_EX_WITH_ATOMIC_CMP_AND_SWP]:
+    if opcode in [ibv_wr_opcode.IBV_WR_ATOMIC_CMP_AND_SWP,
+                  ibv_wr_opcode.IBV_QP_EX_WITH_ATOMIC_CMP_AND_SWP]:
```

Comment:
ibv_qp_create_send_ops_flags.IBV_QP_EX_WITH_ATOMIC_CMP_AND_SWP
Need to import ibv_qp_create_send_ops_flags